[2024-01-05T15:10:00.447+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=User_counter/run_id=scheduled__2024-01-05T15:09:00+00:00/task_id=PostgresTable permission to 509
[2024-01-05T15:10:00.449+0000] {logging_mixin.py:188} INFO - Failed to change /opt/***/logs/dag_id=User_counter/run_id=scheduled__2024-01-05T15:09:00+00:00/task_id=PostgresTable permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=User_counter/run_id=scheduled__2024-01-05T15:09:00+00:00/task_id=PostgresTable'
[2024-01-05T15:10:00.488+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=User_counter/run_id=scheduled__2024-01-05T15:09:00+00:00/task_id=PostgresTable permission to 509
[2024-01-05T15:10:00.488+0000] {logging_mixin.py:188} INFO - Failed to change /opt/***/logs/dag_id=User_counter/run_id=scheduled__2024-01-05T15:09:00+00:00/task_id=PostgresTable permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=User_counter/run_id=scheduled__2024-01-05T15:09:00+00:00/task_id=PostgresTable'
[2024-01-05T15:10:00.538+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: User_counter.PostgresTable scheduled__2024-01-05T15:09:00+00:00 [queued]>
[2024-01-05T15:10:00.551+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: User_counter.PostgresTable scheduled__2024-01-05T15:09:00+00:00 [queued]>
[2024-01-05T15:10:00.551+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 2
[2024-01-05T15:10:00.572+0000] {taskinstance.py:2192} INFO - Executing <Task(PostgresOperator): PostgresTable> on 2024-01-05 15:09:00+00:00
[2024-01-05T15:10:00.576+0000] {standard_task_runner.py:60} INFO - Started process 424 to run task
[2024-01-05T15:10:00.580+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'User_counter', 'PostgresTable', 'scheduled__2024-01-05T15:09:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/User_counter.py', '--cfg-path', '/tmp/tmp8xliiamn']
[2024-01-05T15:10:00.618+0000] {standard_task_runner.py:88} INFO - Job 11: Subtask PostgresTable
[2024-01-05T15:10:00.727+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=User_counter/run_id=scheduled__2024-01-05T15:09:00+00:00/task_id=PostgresTable permission to 509
[2024-01-05T15:10:00.727+0000] {logging_mixin.py:188} INFO - Failed to change /opt/***/logs/dag_id=User_counter/run_id=scheduled__2024-01-05T15:09:00+00:00/task_id=PostgresTable permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=User_counter/run_id=scheduled__2024-01-05T15:09:00+00:00/task_id=PostgresTable'
[2024-01-05T15:10:00.730+0000] {task_command.py:423} INFO - Running <TaskInstance: User_counter.PostgresTable scheduled__2024-01-05T15:09:00+00:00 [running]> on host 8209e4f5b06e
[2024-01-05T15:10:00.834+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='User_counter' AIRFLOW_CTX_TASK_ID='PostgresTable' AIRFLOW_CTX_EXECUTION_DATE='2024-01-05T15:09:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-05T15:09:00+00:00'
[2024-01-05T15:10:00.836+0000] {sql.py:276} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS pet (
            pet_id SERIAL PRIMARY KEY,
            name VARCHAR NOT NULL,
            pet_type VARCHAR NOT NULL,
            birth_date DATE NOT NULL,
            OWNER VARCHAR NOT NULL);
          
[2024-01-05T15:10:00.851+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2024-01-05T15:10:00.865+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2024-01-05T15:10:00.870+0000] {sql.py:432} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS pet (
            pet_id SERIAL PRIMARY KEY,
            name VARCHAR NOT NULL,
            pet_type VARCHAR NOT NULL,
            birth_date DATE NOT NULL,
            OWNER VARCHAR NOT NULL);
          , parameters: None
[2024-01-05T15:10:00.894+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=User_counter, task_id=PostgresTable, execution_date=20240105T150900, start_date=20240105T151000, end_date=20240105T151000
[2024-01-05T15:10:00.922+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-05T15:10:00.952+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
